wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -841,6 +869,10 @@ extension ZMUserSession: ZMSyncStateDelegate {
         fetchFeatureConfigs()
         recurringActionService.performActionsIfNeeded()
 
+        Task {
+            await self.cRLsChecker.checkExpiringCRLs()
+        }
```

Comment:
Should the task be stored in a var and cancelled if the object deallocates?

Reply:
discussed 1:1 and agreed to leave it as is